<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorTrajectoryPartsWidget</class>
 <widget class="QWidget" name="EditorTrajectoryPartsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorTrajectoryPartsWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,3" columnminimumwidth="0,0">
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="newNameLineEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>New trajectory part name...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="moveUpButton">
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="moveDownButton">
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Point radius:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Line width:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="pointRadiusSpinBox"/>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="lineWidthSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onItemPressed(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveUpButton</sender>
   <signal>clicked()</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onMoveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveDownButton</sender>
   <signal>clicked()</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onMoveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointRadiusSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onRadiusPointChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EditorTrajectoryPartsWidget</receiver>
   <slot>onLineWidthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onItemPressed(QModelIndex)</slot>
  <slot>onAdd()</slot>
  <slot>onRemove()</slot>
  <slot>onMoveUp()</slot>
  <slot>onMoveDown()</slot>
  <slot>onRadiusPointChanged(int)</slot>
  <slot>onLineWidthChanged(int)</slot>
 </slots>
</ui>
